packages/cilium/package.yaml

```diff
@@ -1,5 +1,5 @@
 url: https://helm.cilium.io/cilium-1.9.6.tgz
-packageVersion: 04
```

Comment:
I don't think you can revert the package versions - the versions always need to move forward, even if you're reverting a change to the chart.

Reply:
ok, let me change that